docs/utils.rst

```diff
@@ -158,6 +158,18 @@ sgtk.util.json
 .. autofunction:: loads
 
 
+.. currentmodule:: sgtk.util.pickle
+
+
+sgtk.util.pickle
+-----------------------------------
+
+.. autofunction:: dump
+.. autofunction:: dumps
+.. autofunction:: load
+.. autofunction:: loads
```

Comment:
I am not exposing the env var stuff here, as those methods are more for internal consumption I think for now.